content/applications/sales/subscriptions/closing.rst

```diff
@@ -1,105 +1,114 @@
-   dissatisfied customers, and who retain their customers by making it easier for them to leave if
-   they want to. By giving them the opportunity to close their own subscriptions, your customers do
-   not feel trapped because they subscribed to your products/services.
-
-In summary, the **Odoo Subscriptions** application gives you the
-**possibility to choose what you want to apply**. Indeed, you can decide whether to give your customers the
-option to close their subscriptions whenever they want to or to restrict this possibility. It depends
-on you and we will show you how to do that in our amazing application.
+Odoo *Subscriptions* provides flexibility for businesses to decide whether customers can self-manage
+their subscriptions, or restrict that ability entirely.
 
 Configuration
 =============
 
-Go to :menuselection:`Subscriptions --> Configuration --> Subscription templates`. From there, you can
-create a new *Subscription template* or modify an existing one. When editing your template, underneath
-the Invoicing tab, you have the possibility to activate the option *Closable by customer*.
-Once enabled, this option gives your customers the right to close their own subscriptions.
+To configure recurring plans offered, navigate to :menuselection:`Subscriptions app -->
+Configuration --> Recurring Plans`. From there, either create a new plan by clicking :guilabel:`New`
+or select an existing one to modify it.
 
-.. image:: closing/configuration-to-close-your-subscriptions.png
-  :align: center
-  :alt: Configuration to close your subscription with Odoo Subscriptions
+Once on the :guilabel:`Recurring Plans` form, enable the :guilabel:`Closable` option, in the
+:guilabel:`Self-Service` section, to allow customers to close their own subscriptions using the
+customer portal.
 
-.. note::
-   Be sure to check out our documentation on
-   how to :doc:`Use subscription templates <plans>`
-   to fully understand the importance of this feature in a basic flow using the
-   **Odoo Subscriptions** application.
+.. image:: closing/recurring-plans-closable-option.png
+   :align: center
+   :alt: The Closable option on a recurring plan form in Odoo Subscriptions.
 
-Close your first subscription
-=============================
+.. seealso::
+   :doc:`Configure recurring plans <plans>`
+
+Close a subscription
+====================
 
 Administrator view
 ------------------
 
-Once confirmed, a quotation becomes a sales order and a new subscription is automatically created.
-Therefore, this subscription has the status *In progress*. From there, you have the possibility to
-close the subscription.
+After a quotation for a subscription product has been confirmed, it becomes a sales order, and the
+subscription is officially :guilabel:`In Progress`.
+
+At that point, the ability to close the subscription becomes available, via the :guilabel:`Close`
+button at the top of the :guilabel:`In Progress` subscription order. This option is also available
```

Comment:
"top of the **In Progress** subscription order" didn't make sense to me when I read it a couple times.
Here's a suggested rewrite: 

```suggestion
button at the top of the subscription order, near the row that contains :guilabel:`In Progress` and other stages. This option is also available
```